.github/workflows/full-build.yml

```
@@ -2,9 +2,20 @@ name: Full Build
 
 on:
   push:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'
   pull_request:
+    branches:
+      - main
   release:
     types: [published]
+  workflow_dispatch:
```

```suggestion
  workflow_dispatch
```
It's my first time seeing github workflows, so I did some reading. I think `:` is not necessary since you are not providing any additional inputs.

---

Hi @marek-piirikivi, this didn't pass our yaml pre-commit check, so I had to undo it.